modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/deployer/ObjectDefinitionDeployerImpl.java

```diff
@@ -727,6 +636,151 @@ private void _initSystemObjectDefinition(
 			});
 	}
 
+	private boolean _shouldUnregisterApplication(String restContextPath) {
+		List<String> companyIds = _restContextPathCompanyIds.get(
+			restContextPath);
+
+		if ((companyIds != null) && ListUtil.isNotEmpty(companyIds)) {
```

Comment:
IntelliJ warnings are useful. Seems like `isNotEmpty` also check for nulls, so, the first check is not necessary.
![image](https://user-images.githubusercontent.com/17163902/228632785-87c3cfc2-e476-48bb-8e06-9862d86f4648.png)